lib/io.c

```diff
@@ -21,6 +22,7 @@ void metal_io_init(struct metal_io_region *io, void *virt,
 	io->virt = virt;
 	io->physmap = physmap;
 	io->size = size;
+	io->physmap_drv = physmap_drv;
```

Comment:
it could be the device physmap if used on driver side. to rename (`remote_physmap`?)

Reply:
The difference between io->physmap and io->physmap_drv (or new name) should be documented in comment.